modules/healthconnectonfhir/src/main/java/edu/stanford/healthconnectonfhir/Loinc.kt

```diff
@@ -0,0 +1,20 @@
+package edu.stanford.healthconnectonfhir
+
+object Loinc {
```

Comment:
I would be more in favor of an enum instead `enum class Loinc(code: String)`, and move `COMPONENT` in the `ObservationToRecordMapperImpl` in companion object

Reply:
but don't we then lose the connection of which Loinc code can have which components?